pkg/kubectl/cmd/delete/delete.go

```diff
@@ -33,13 +33,13 @@ import (
 	"k8s.io/cli-runtime/pkg/genericclioptions/resource"
 	"k8s.io/client-go/dynamic"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
-	kubectlwait "k8s.io/kubernetes/pkg/kubectl/cmd/wait"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/wait"
```

Comment:
`cmdwait` will be explicit which wait it it, we have at least 3 other wait packages I'm aware of, so let's be clear with imports when we can.